webpack/assets/javascripts/react_app/components/ChartBox/ChartBox.js

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card, Modal } from 'patternfly-react';
+import { isEqual } from 'lodash';
+import classNames from 'classnames';
+import { bindMethods } from '../../common/helpers';
+import DonutChart from '../common/charts/DonutChart';
+import BarChart from '../common/charts/BarChart';
+import { navigateToSearch } from '../../../services/charts/DonutChartService';
+import Loader from '../common/Loader';
+import MessageBox from '../common/MessageBox';
+import { translate as __ } from '../../common/I18n';
+
+class ChartBox extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { showModal: false };
+    bindMethods(this, ['onClick', 'closeModal', 'openModal']);
```

Comment:
we should avoid using `bindMethods`,
can you change `'onClick', 'closeModal', 'openModal'` into arrow functions?

Reply:
Removed onClick as it just calls openModal, and changed them into arrow functions